modules/unsupported/csv/src/test/java/org/geotools/data/csv/CSVWriteTest.java

```diff
@@ -307,8 +306,11 @@ public void appendContent() throws Exception {
         String contents2 = new String(baos2.toByteArray(), StandardCharsets.UTF_8);
 
         // Making sure whitespace doesn't cause problems
+        // and adjust for potential windows line endings.
         contents1 = contents1.replace(" ", "").trim();
         contents2 = contents2.replace(" ", "").trim();
+        contents1 = contents1.replace("\r", "");
+        contents2 = contents2.replace("\r", "");
```

Comment:
I think I would make an assertEqualsIgnoreWhitespace method, that takes care of any whitespace (not just special casing " "  and "\n") and then we can use it more generally.

Rather than hold up the show let me do that when merging.